### PR TITLE
Data model creation

### LINKED ISSUE #19: Add package to create model
## User Story
**As an** Admin, **I want to** be able to deploy a model, **so that** it can be used for contributions

## Gherkins
- **Given** all packages are installed
  - **When** the admin executes the deployment command
  - **Then** it creates the model, deploys it to the Ceramic node and exports it in `.JSON` format

### PR DESCRIPTION
**Pull request by Zitzak**
## What has been done?
- Add package to manage the creation of the data model
- Add script to create the data model
- Add Readme

## Implemented
- closes #19

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,13 +5,16 @@
   "scripts": {
     "lint": "eslint .",
     "format": "prettier --write .",
+    "build:off-chain": "yarn workspace off-chain run build",
```
Comment (codebuster22) [inline package.json]: I think you might have used an old package.json file.
The workspace name has changed. It is now `bacalhau`

File: tsconfig.json [hunk 2]
```diff
@@ -4,6 +4,9 @@
       "path": "./packages/on-chain"
     },
     {
+      "path": "./packages/off-chain"
```
Comment (codebuster22) [inline tsconfig.json]: this is not a package anymore.

**Review by codebuster22 — CHANGES_REQUESTED** [review]
everything looks good to me.
only two comments

**Review by fabianschu — APPROVED** [review]
lgtm

**Review by codebuster22 — APPROVED** [review]
lgtm

**Pull request merged**